docker-compose.yml

```diff
@@ -4,15 +4,19 @@ services:
     image: postgres:10-alpine
     environment:
       - POSTGRES_USER=root
+    privileged: true
```

Comment:
when I talked to Ethan I'm pretty sure he said these docker-compose changes shouldn't affect Mac